live/liveview.go

```diff
@@ -393,10 +504,10 @@ func (s *socket) dispatch(ctx context.Context, msg *phx.Msg) ([]byte, error) {
 		return phx.NewReplyDiff(*msg, diff).JSON()
 	case "live_patch":
 		r := s.req.Clone(s.req.Context()) // todo: background context?
-		v, err := s.config.viewForRequest(r, s.view)
+		v, code, _ := s.config.viewForRequest(nil, r, s.view)
 		s.view = v // update the view
-		if err != nil {
-			return nil, err
+		if code%100 == 5 {
+			return nil, fmt.Errorf("Status code 500 patching LiveView in %v", r.URL)
```

Comment:
nit:  error messages in go by convention start with a lower case letter, so: "status ..."